message-bridge/src/main/resources/application.properties

```diff
@@ -0,0 +1,34 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+ibm.mq.host=localhost
+ibm.mq.port=1414
+ibm.mq.channel=DEV.APP.SVRCONN
+ibm.mq.queueManagerName=QM1
+ibm.mq.user=app
+ibm.mq.password=passw0rd
+ibm.mq.queue=DEV.QUEUE.1
+
+quarkus.artemis.amqConnectionFactory.url=tcp://localhost:61616
+quarkus.artemis.amqConnectionFactory.username=admin
+quarkus.artemis.amqConnectionFactory.password=admin
+amq.queue=in
+
+quarkus.camel.servlet.url-patterns=/*
+
+quarkus.pooled-jms.transaction=xa
+quarkus.pooled-jms.pooling.enabled=true
+quarkus.pooled-jms.max-connections=8
```

Comment:
I'd like to add `quarkus.transaction-manager.enable-recovery=true` and it could be helpful to test the crash crash secenario.

Reply:
thanks, will check that out as well